README.md

```diff
@@ -130,6 +132,18 @@ Shipwright supports any tool that can build container images in Kubernetes clust
 
 - After your *BuildRun* is completed, check your container registry, you will find the new generated image uploaded there.
 
+<a name="running-on-openshift"></a>
```

Comment:
What is the reason for this? Markdown renderers usually create this automatically.

Reply:
Sorry, I was not aware that Markdown renders automatically. Fixed the same.